modules/RTC/TPCUtils.js

```diff
@@ -942,7 +946,10 @@ export class TPCUtils {
 
             if (isSender && mLine.ext?.length) {
                 const headerIndex = mLine.ext.findIndex(ext => ext.uri === DD_HEADER_EXT_URI);
-                const shouldNegotiateHeaderExts = codec === CodecMimeType.AV1 || codec === CodecMimeType.H264;
+                const shouldNegotiateHeaderExts = codec === CodecMimeType.AV1 || codec === CodecMimeType.H264
+
+                    // Removing DD ext header lines from the sdp breaks the scalability for FF with version 136+.
+                    || (codec === CodecMimeType.VP8 && browser.isFirefox());
```

Comment:
Should we check for the FF version?

Reply:
No need. This function will be called Only for FF 136+ anyway because the previous versions do not support DD.

Reply:
We should get rid of this method all together since the bridge now gets scalability info from DD for all the video codecs. Earlier it used to parse them only for AV1 and H.264. Once DD is offered by Jicofo and negotiated by the  client, I see no reason to remove it from the remote desc. @JonathanLennox pls correct me if I am wrong.

Reply:
Awesome! 

I suggest to remove it as a separate PR.